### PR TITLE
Make compound absorber radius fit bacteria size

### PR DESCRIPTION
**Pull request by Dak2896**
- Bacteria absorber radius fit visual size
#755

### CODE REVIEW

File: scripts/microbe_stage/microbe_editor/microbe_editor.as [hunk 1]
```diff
@@ -960,6 +966,27 @@ class MicrobeEditor{
...
 
+            // Grab render of player cell
+            auto node =  world.GetComponent_RenderNode(player);          
+
+
+            // Grab memnbrane to update scale value for absorber compound system
+            auto membraneComponent = world.GetComponent_MembraneComponent(player);
+
+            //! Change player species cell size depending on presence or not of nucleus
+            if(checkIsNucleusPresent()) {
```
Comment (Untrustedlife) [inline scripts/microbe_stage/microbe_editor/microbe_editor.as]: What is this for? Bacteria size is already correct, whats the point of this change???

Comment (Dak2896) [reply scripts/microbe_stage/microbe_editor/microbe_editor.as]: is not already merged in master branch.

**Comment by hhyyrylainen** [comment]
This needs to wait until 0.4.1 is done before merging.

**Comment by Dak2896** [comment]
Now this has some conflicts, better if i close pull and open new one?

**Comment by hhyyrylainen** [comment]
You can just push updates. This will be squash merged so it doesn't matter if this contains "random" commits in it as long as the final diff shows everything is good.

**Comment by Untrustedlife** [comment]
Looks good, looks like there is a conflict though

**Comment by Dak2896** [comment]
So we can close this right?

**Comment by hhyyrylainen** [comment]
Yes. Though I'm not sure which approach (scaling the membrane's reported size or having a scale directly in the absorber) is better in the long run. I can see benefits in both depending on what kind of features we'll be adding in the future.

**Pull request closed without merging**